br/pkg/backup/push.go

```diff
@@ -168,13 +168,16 @@ func (push *pushDown) pushBackup(
 						logutil.CL(ctx).Error("", zap.String("error", berrors.ErrKVStorage.Error()+": "+errMsg),
 							zap.String("work around", "please ensure br and tikv node share a same disk and the user of br and tikv has same uid."))
 					}
-
 					if utils.MessageIsPermissionDeniedStorageError(errPb.GetMsg()) {
 						errMsg := fmt.Sprintf("I/O permission denied error occurs on TiKV Node(store id: %v; Address: %s)", store.GetId(), redact.String(store.GetAddress()))
 						logutil.CL(ctx).Error("", zap.String("error", berrors.ErrKVStorage.Error()+": "+errMsg),
 							zap.String("work around", "please ensure tikv has permission to read from & write to the storage."))
 					}
-					return res, berrors.ErrKVStorage
+					return res, errors.Annotatef(berrors.ErrKVStorage, "unknown error in store %v at %s: %s",
```

Comment:
maybe it's not a unknown error, since the above error check just log only

Reply:
nice catch...